packages/token-balance-cache/test/tbc.spec.ts

```diff
@@ -120,4 +120,32 @@ describe('TBC unit tests', () => {
     });
     tbc.close();
   });
+
+  it('should return token balances for erc1155 token id', async () => {
```

Comment:
can we add some more tests for each of the new BPs? now that we have 4 different eth providers (ERC20, ERC721, ERC1155, and base token), we should have sufficient coverage there to know that they 1. work, 2. fail when we expect them to!